yt/frontends/ramses/fields.py

```diff
@@ -241,35 +256,83 @@ def create_cooling_fields(self):
         filename = "%s/cooling_%05i.out" % (
             os.path.dirname(self.ds.parameter_filename), int(num))
 
-        if not os.path.exists(filename): return
-        def _create_field(name, interp_object):
+        if not os.path.exists(filename): 
+            print("This output has no cooling fields")
```

Comment:
It would be better here to issue a warning instead. E.g.:
```
from yt.funcs import mylog
[...]
mylog.warning('This output has no cooling fields')
```